src/couch_views/src/couch_views_util.erl

```diff
@@ -327,6 +328,33 @@ active_tasks_info(ChangesDone, DbName, DDocId, LastSeq, DBSeq) ->
     }.
 
 
+maybe_disable_custom_reduce_funs(Views) ->
+    case config:get_boolean("couch_views", "custom_reduce_enabled", false) of
+        true ->
+            Views;
+        false ->
+            disable_custom_reduce_funs(Views)
+    end.
+
+
+disable_custom_reduce_funs(Views) ->
+    lists:map(fun(View) ->
+        #mrview{
+            reduce_funs = ReduceFuns
+        } = View,
+        {Builtin, Custom} = lists:partition(fun({_Name, RedSrc}) ->
+            case RedSrc of
+                <<"_", _/binary>> -> true;
```

Comment:
This isn't very strong, it would allow `_foo = 12; fun(keys, values, rereduce) {}` through and I think we'd run that? I think we need to match the whole thing with a regex like `^_[a-z]+]$`

Reply:
That is to say, I don't think this PR is the place to change the definition of a builtin reduce. Even if a regexp or even a list membership for known builtins would be more useful from the user experience point of view.